### PR TITLE
Netsim filter router logs

### PR DESCRIPTION
**Pull request by Hamms**
PD needs the ability to see logs for just the connected router. Rather than restoring the log table and losing the ability to ignore logs most of the time, I added a simple toggle to the log modal that lets us switch between filtered and unfiltered view.

There are two things that influence when you can switch: whether or not you're connected to a router and whether or not you are in a level with connected routers.
- If you are not connected to a router, you can only see logs for all the routers.
- If you are connected to a router
  - and the level has connected routers, you can switch between the modes.
  - and the level does not have connected routers, you can only view the router you are connected to.

### CODE REVIEW

File: apps/i18n/netsim/en_us.json [hunk 1]
```diff
@@ -54,6 +54,8 @@
   "logBrowserHeader": "All Router Logs",
   "logBrowserHeader_all": "All Router Logs",
   "logBrowserHeader_mine": "My Router Logs",
+  "logBrowserHeader_toggle_all": "show all routers",
+  "logBrowserHeader_toggle_mine": "show my router",
```
Comment (islemaster) [inline apps/i18n/netsim/en_us.json]: Should these be capitalized/sentence-cased?

**Comment by islemaster** [comment]
Screenshot?

File: apps/i18n/netsim/en_us.json (hunk 1 above)
Comment (Hamms) [inline apps/i18n/netsim/en_us.json]: Oh, definitely

**Comment by Hamms** [comment]
Screenshots!

All routers:

image

Single router:

image

**Comment by islemaster** [comment]
Oh... I meant sentence-casing the actual localization string content ("Show my router"/"Show all routers"), not the variable name.  That's a good change too, though.

**Comment by islemaster** [comment]
LGTM! :+1:

**Comment by Hamms** [comment]
Oh, ha. I could do that, but aesthetically I prefer all lower-case for a toggle operation like that.

**Pull request merged**